repos/spack_repo/builtin/packages/py_numba/package.py

```diff
@@ -36,6 +36,8 @@ class PyNumba(PythonPackage):
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
 
+    depends_on("rocm-openmp-extras", when="%llvm-amdgpu", type="build")
```

Comment:
If this needs to be added everywhere `llvm-amdgpu` is needed, we should fix it in the compiler package. Is there some issue already open about `llvm-amdgpu` + OpenMP, with `llvm-amdgpu` maintainers pinged?